.github/workflows/cd.yml

```diff
@@ -16,12 +16,12 @@ jobs:
     name: Authorize
     runs-on: ubuntu-latest
     steps:
-    - name: ${{ github.actor }} permission check to do a release
-      uses: "lannonbr/repo-permission-check-action@2.0.2"
-      with:
-        permission: "write"
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: ${{ github.actor }} permission check to do a release
+        uses: 'lannonbr/repo-permission-check-action@2.0.2'
+        with:
+          permission: 'write'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Where we use the GITHUB_TOKEN?

Reply:
`GITHUB_TOKEN` is required by `lannonbr/repo-permission-check-action` based on this doc: https://github.com/lannonbr/repo-permission-check-action#usage